src/components/Results.vue

```diff
@@ -232,4 +233,7 @@
     width: 100%;
     height: 100%;
   }
+  svg {
+    width: 75%;
```

Comment:
Nie ustawiałbym globalnie szerokości dla SVG, nawet jeśli to jest per komponent.
Może dać temu SVG z tarczą jakiś id np. `armor` i użyć tutaj `#armor { ... }` zamiast `svg { ... }`?

Reply:
Mi już w krew weszło takie pisanie, ale faktycznie może być mylące. Już wrzucam commita.